config/docker/externalJs/loadTags.js

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+console.log('Custom tags loaded');
```

Comment:
To be consistent with other logs : `console.log(new Date().toISOString(), 'Custom tags loaded')`